src/components/action-group/action-group.tsx

```diff
@@ -59,6 +59,8 @@ export class ActionGroup
 
   /**
    * Indicates the layout of the component.
+   *
+   * @deprecated Use the `layout` property on the component's parent instead.
```

Comment:
There is the "grid" mode on the action-group, which currently isn't on the parent for some reason. I'm not really familiar with how those modes interact, but should "grid" be added to the parent `action-bar` layout as a value?

Reply:
I think the grid layout is just for the action-pad so I think as it is makes sense.